projects/canopy-test-app/src/app/app.component.html

```diff
@@ -287,6 +287,22 @@
               <lg-radio-button value="blue">Blue</lg-radio-button>
             </lg-radio-group>
 
+            <lg-filter-group formControlName="filter">
+              Test filter group select one
+              <lg-filter-button value="red">Red</lg-filter-button>
+              <lg-filter-button value="yellow">Yellow</lg-filter-button>
+              <lg-filter-button value="green">Green</lg-filter-button>
+              <lg-filter-button value="blue">Blue</lg-filter-button>
+            </lg-filter-group>
+
+            <lg-filter-group variant="select-multiple" valueArrayName="filters" formArrayName="filters">
```

Comment:
In my head I expected this to be like the below, but I haven't at this stage done any background research or really looked into it, so I may be talking nonsense, I'll have a read to get my head around it more. 
```
<lg-filter-group variant="select-multiple" formControlName="filter">
```

Reply:
to get the values in an array, like ['redFilter', 'greenFilter'], we need to use a control container, either a form group or a form array, so we can contain more controls.The FormControlName, on the filter-group component, is only supported with radio buttons as they change the same value on the same control, as opposed to pushing and removing values.